File: go/long_live.go
package main
import (
	"fmt"
	"strconv"
	"strings"
	"time"
)

func main() {
	start := time.Now()
	var table []string
	for i := 0; i < 200000; i++ {
		table = append(table, strings.Repeat("value-", 2) + strconv.Itoa(i))
	}
	for j := 0; j < 400000; j++ {
		_ = strings.Repeat("scratch-", 2) + strconv.Itoa(j)
	}
	fmt.Println("long_live", len(table))
	fmt.Printf("Time: %v\n", time.Since(start))
}

File: python/long_live.py
import gc
import time
import gcstats  # noqa: F401  (registers stats at exit)

start = time.perf_counter()
table = []
for i in range(200000):
    table.append("value-" * 2 + str(i))
for j in range(400000):
    _ = "scratch-" * 2 + str(j)
elapsed = time.perf_counter() - start
print("long_live", len(table))
print("Time:", f"{elapsed:.3f}", "seconds")